tests/lib/rules/prefer-destructuring.js

```diff
@@ -35,39 +35,61 @@ ruleTester.run("prefer-destructuring", rule, {
 
             // Ensure that the default behavior does not require desturcturing when renaming
             code: "var foo = object.bar;",
+            options: [{ VariableDeclarator: { object: true } }]
+        },
+        {
+
+            // Ensure that the default behavior does not require desturcturing when renaming
+            code: "var foo = object.bar;",
             options: [{ object: true }]
         },
         {
             code: "var foo = object.bar;",
+            options: [{ VariableDeclarator: { object: true } }, { enforceForRenamedProperties: false }]
+        },
+        {
+            code: "var foo = object.bar;",
             options: [{ object: true }, { enforceForRenamedProperties: false }]
         },
         {
             code: "var foo = object['bar'];",
-            options: [{ object: true }, { enforceForRenamedProperties: false }]
+            options: [{ VariableDeclarator: { object: true } }, { enforceForRenamedProperties: false }]
         },
         {
-            code: "var foo = object[bar];",
+            code: "var foo = object['bar'];",
```

Comment:
Is this change necessary? (I don't mind the new version, I'm just sometimes worried about changing old tests because that can prevent regressions from being caught.)